esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -749,13 +749,39 @@ IEspDFUWorkunit *CFileSprayEx::createDFUWUFromSashaListResult(const char *result
             case 5:
                 const char *commandStr = wuInfoFields.item(5);
                 if (!isEmptyString(commandStr))
-                    dfuWU->setCommand(decodeDFUcommand(commandStr));
+                    setDFUCommand(commandStr, dfuWU);
                 break;
         }
     }
     return dfuWU.getClear();
 }
 
+void CFileSprayEx::setDFUCommand(const char *commandStr, IEspDFUWorkunit *dfuWU)
+{
+    bool isNumber = true;
```

Comment:
Probably overkill to implement both.  I would delete lines 761-777 and it should behave exactly the same.

Reply:
The lines have been deleted.